### DISCUSSION TITLE
waiting for status to be reported

### DISCUSSION THREAD

**Discussion by prafulbbandre** [oppia/oppia-android#4935] [Q&A (Other)] [answered]
i had open a pr for updating readme.md file.
but the checks are not running and it says waiting for status to be reported.
![Screenshot from 2023-04-01 00-06-58](https://user-images.githubusercontent.com/109658545/229202689-e256c314-64bc-42be-b765-28f5f63da3a1.png)

**Comment by MohitGupta121** [comment, marked as answer]
@prafulbbandre Thanks you for PR, please be patient oppia maintainer trigger the workflows for new contributors after review. 
Thanks, please feel free to ask any other questions.
